### PR TITLE
os/tools/mkbinheader : Add ram information support

### PR DESCRIPTION
**Pull request by YiHyunJin**
Added ram information in binary header.
It is used to allocate ram partition when binary manager loads binary.

### CODE REVIEW

File: os/tools/mkheader.py [hunk 1]
```diff
@@ -23,10 +23,10 @@
 #
 # header information :
 #
-# / header size (2byte) / binary type (2byte) / binary size (4byte)
-# / Binary name (16byte) / Binary version (16byte)
+# / Header size (2byte) / Binary type (2byte) / Binary size (4byte)
+# / Binary name (16byte) / Binary version (16byte) / Binary ram (4byte)
```
Comment (jeongarmy) [inline os/tools/mkheader.py]: Binary ram size

**Comment by jeongarmy** [comment]
please update commit message.
Added ram information in header.
It is used to allocate ram partition when binary manager loads binary.

File: os/tools/mkheader.py (hunk 1 above)
Comment (YiHyunJin) [reply os/tools/mkheader.py]: okay I fixed.

File: os/tools/mkheader.py [hunk 2]
```diff
@@ -43,6 +44,7 @@
 kernel_ver = sys.argv[3]
 binary_name = sys.argv[4]
 binary_ver = sys.argv[5]
+binary_ram = sys.argv[6]
```
Comment (jeongarmy) [inline os/tools/mkheader.py]: please change 'ram' to 'ram_size', binary_ram_size and BINARY_RAM_SIZE_S

File: os/tools/mkheader.py [hunk 3]
```diff
@@ -23,10 +23,10 @@
 #
 # header information :
 #
-# / header size (2byte) / binary type (2byte) / binary size (4byte)
-# / Binary name (16byte) / Binary version (16byte)
+# / Header size (2byte) / Binary type (2byte) / Binary size (4byte)
+# / Binary name (16byte) / Binary version (16byte) / Binary ram size (4byte)
 # / Kernel version (8byte) / Jump address (4byte)
-# header size is 52byte.
+# header size is 56byte.
```
[the remark this answers is not in the document]
Comment (YiHyunJin) [reply os/tools/mkheader.py]: okay, Thanks.

File: os/tools/mkheader.py (hunk 2 above)
Comment (YiHyunJin) [reply os/tools/mkheader.py]: I modified.

File: os/tools/mkheader.py [hunk 4]
```diff
@@ -23,10 +23,10 @@
 #
 # header information :
 #
-# / header size (2byte) / binary type (2byte) / binary size (4byte)
-# / Binary name (16byte) / Binary version (16byte)
-# / Kernel version (8byte) / Jump address (4byte)
-# header size is 52byte.
+# / Header size (2bytes) / Binary type (2bytes) / Binary size (4bytes)
```
Comment (sunghan-chang) [inline os/tools/mkheader.py]: Could you use Jeongchan's way to show packet information as shown below
```
	 * +------------------------------------------+
	 * | sender_pid | msg type |     message      |
	 * | (4bytes)   | (4bytes) | (Max 65527bytes) |
	 * +------------------------------------------+
```

**Comment by sunghan-chang** [comment]
@YiHyunJin This commit does not "add ram information in binary header". It adds ram information support in binary header.

File: os/tools/mkheader.py (hunk 4 above)
Comment (YiHyunJin) [reply os/tools/mkheader.py]: I modified.

File: os/tools/mkbinheader.py [hunk 5]
```diff
@@ -35,6 +40,7 @@
 # argv[3] is kernel version.
 # argv[4] is binary name.
 # argv[5] is binary version.
+# argv[6] is binary ram size.
```
Comment (sunghan-chang) [inline os/tools/mkbinheader.py]: # argv[6] is a ram size required to run this binary.

**Pull request merged**